recipes/hash-library/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,29 @@
+#include "crc32.h"
+#include "keccak.h"
+#include "md5.h"
+#include "sha1.h"
+#include "sha256.h"
+#include "sha3.h"
```

Comment:
Should we install the headers into a subfolder `hash-library` to avoid potential naming clashes with other headers?

Reply:
We follow the upstream approach. What you can is create hash-library folder, but include it includedir. As example:

`self.cpp_info.includedirs = ["include", os.path.join("include", "hash-library")]`